spinoffs/Affably_Evil/scenarios/Strych.cfg

```diff
@@ -561,6 +561,7 @@
                         message=_"I will be on my way."
                     [/message]
                     {AE_TRANSITION 14-30 23-25 Southern_Estates _"Southern Estates" 24 23}
+[show_objectives][/show_objectives]
```

Comment:
Bad indentation.

Reply:
There's a bug in the install when building from source, and I've had several wesnoth versions on that box.  Who knows which wmlindent is installed.  I'll check.

Or maybe I just missed running it on one.  More likely.